src/web/mjs/engine/Settings.mjs

```diff
@@ -253,6 +253,20 @@ export default class Settings extends EventTarget {
             ],
             value: 'none'
         };
+
+        this.NovelDarkmode = {
+            label: 'Light/Dark mode for Novels',
+            description: [
+                'choose between light and dark mode for novels',
```

Comment:
Start with a capital letter
Also a bit misleading, `mode` implies that it is a setting for the reader, that can be changed as the iser pleases, but in fact it determines the colorset in which the images are downloaded (once downloaded in a mode, it cannot be changed afterwards without re-downloading)

Reply:
what would you suggest instead of mode
"dark mode" is synonymous with dark background and light text

Reply:
Maybe something like this:
`Check this option to download novels in dark mode (inverted colors) instead of their original colors.`